pkg/distro/generic/images.go

```diff
@@ -345,6 +345,15 @@ func installerCustomizations(t *imageType, c *blueprint.Customizations) (manifes
 		if isoboot := installerConfig.ISOBootType; isoboot != nil {
 			isc.ISOBoot = *isoboot
 		}
+
+		// Put the kickstart file in the root of the iso, some image
+		// types (like rhel10/image-installer) put them there, some
+		// others (like fedora/image-installer) do not and because
+		// its not uniform we need to make it configurable.
+		// XXX: unify with rhel-11 ? or rhel-10.x?
```

Comment:
Let's turn this into an issue/task so we don't forget about it

Reply:
Sure.

Reply:
https://github.com/osbuild/images/issues/1761